tests/cognito/package.json

```diff
@@ -9,28 +9,24 @@
     "watch": "tsc -w",
     "test": "if [ -f outputs.json ]; then jest; else echo \"ERROR: Deploy stack with AWS resources first: cdk deploy -O outputs.json --toolkit-stack-name AwsJwtVerifyTest-toolkit\"; exit 1; fi",
     "cdk": "cdk",
-    "postinstall": "cd lib/lambda-authorizer && npm install --no-save --force --no-package-lock ../../../../aws-jwt-verify.tgz"
+    "postinstall": "npm install --no-save --force --no-package-lock ../../aws-jwt-verify.tgz && cd lib/lambda-authorizer && npm install --no-save --force --no-package-lock ../../../../aws-jwt-verify.tgz"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.157.0",
-    "@types/jest": "^27.4.1",
-    "@types/node": "17.0.21",
-    "aws-cdk": "1.157.0",
-    "jest": "^27.5.1",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.5.0",
-    "typescript": "~4.5.5"
+    "@types/jest": "^29.5.12",
+    "@types/jsdom": "^21.1.7",
+    "@types/node": "22.5.4",
+    "aws-cdk": "2.164.1",
```

Comment:
Do we want to pin to 2.164.1?  It doesn't affect the library usage, just our tests - but maybe most recent version is better.

```
****************************************************
*** Newer version of CDK is available [2.173.4]  ***
*** Upgrade recommended (npm install -g aws-cdk) ***
****************************************************
```